src/diagram/components/variable-form.tsx

```diff
@@ -1,63 +1,62 @@
 import { observer } from "mobx-react-lite";
-import { Instance } from "mobx-state-tree";
 import React from "react";
-import { DQNode, Operation } from "../models/dq-node";
+import { Operation, VariableType } from "../models/variable";
 
 interface IProps {
-    node: Instance<typeof DQNode>;
-  }
+    variable: VariableType;
+}
   
-const _NodeForm: React.FC<IProps> = ({ node }) => {
+const _VariableForm: React.FC<IProps> = ({ variable }) => {
   const onValueChange = (evt: any) => {
     // if the value is null or undefined just store undefined
     if (evt.target.value == null) {
-      node.setValue(undefined);
+      variable.setValue(undefined);
     } else {
-      node.setValue(parseFloat(evt.target.value));
+      variable.setValue(parseFloat(evt.target.value));
```

Comment:
Rather than calling `variable.setValue` twice, I would use something like:
```suggestion
      const newValue = evt.target.value != null ? parseFloat(evt.target.value) : undefined;
      variable.setValue(newValue);
```
and the same for unit, name, and operation below.

Reply:
I kind of prefer the original form since it would be easier to debug being on multiple lines. It also helps with code coverage stats. I really wish there was something like an `if` that returned a value. I could do a anonymous function like:
```
const newValue = (function(){
  if (evt.target.value != null) {
    return parseFloat(evt.target.value)
  }
})();
variable.setValue(newValue);
```
But that doesn't look much better to me than just calling setValue twice.